README.adoc

```diff
@@ -4002,6 +4002,30 @@ format('Hello, %{name}', name: 'John')
 format('Hello, %<name>s', name: 'John')
 ----
 
+=== Long Strings [[heredoc-long-strings]]
+
+Break long strings into multiple lines but don't concatenate them with `+`.
+If you want to add newlines, use `Heredoc`. Otherwise use `\`:
+
+[source,ruby]
+----
+# bad
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. " +
+"Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, " +
+"when an unknown printer took a galley of type and scrambled it to make a type specimen book."
+
+# good
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. "\
+"Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, "\
+"when an unknown printer took a galley of type and scrambled it to make a type specimen book."
```

Comment:
It will be easier to understand that `# good` examples are arranged in the order of `/`, heredoc as in the order of explanation.